proof/crefine/RISCV64/VSpace_C.thy

```diff
@@ -251,7 +251,7 @@ lemma handleVMFault_ccorres:
     prefer 3 apply simp
    apply (simp add: handleVMFault_def handleVMFault'_def liftE_bindE condition_const
                     ucast_ucast_mask bind_assoc)
-   apply (rule corres_split[OF _ read_sbadaddr_ccorres[ac]], drule sym, clarsimp)
+   apply (rule corres_split[OF _ read_stval_ccorres[ac]], drule sym, clarsimp)
```

Comment:
Practicing using the github interface... do you remember what the `[ac]` attribute does?

Reply:
IIRC, that's a gadget to convert a `ccorres` rule to a `corres` rule via AutoCorres.